client/setup.py

```diff
@@ -36,6 +36,7 @@
     "GitPython>=3.1.24",
     "filelock",
     "fastapi",
+    "pydantic<=1.10.11",
```

Comment:
pydantic req limitation is in the line-46

```
"pydantic<2.0.0", # current broker and fastapi lib code only work with pydantic < 2.0.0
```

Reply:
Done